### DISCUSSION TITLE
welcome to Effecient Nodes repo

### DISCUSSION THREAD

**Discussion by jags111** [jags111/efficiency-nodes-comfyui#5] [General]
You are welcome to put up any query or discuss issues concerning same and share your feedback so we can collectively solve the same. Look forward to your discussions.

**Comment by pergj** [comment]
When loading the graph, the following node types were not found:
Efficiency Nodes for ComfyUI Version 2.0+[ 🔗](https://github.com/jags111/efficiency-nodes-comfyui)
Nodes that have failed to load will show as red on the graph.

**Comment by DarqueLilly** [comment]
> You are welcome to put up any query or discuss issues concerning same and share your feedback so we can collectively solve the same. Look forward to your discussions.

can u please answer this thread https://github.com/jags111/efficiency-nodes-comfyui/discussions/161  I'm having the same issue
